main.nf

```diff
@@ -458,6 +561,30 @@ if( !params.fasta_index && params.aligner == 'bwameth' ){
     }
 }
 
+/*
+ * PREPROCESSING - Build Biscuit QC assets
+ */
+if( !params.assets_dir &&  params.aligner == 'biscuit' ) {
+    process buildBiscuitQCAssets {
+        tag "$fasta"
+        publishDir path: "${params.outdir}/reference_assets", saveAs: { params.save_reference ? it : null }, mode: params.publish_dir_mode
+
+        input:
+        file fasta from ch_fasta_for_buildBiscuitQCAssets
+
+        output:
+        file "*assets" into ch_assets_dir_for_biscuit_qc, ch_assets_dir_with_cpg_for_epiread
+
+
+        script:
+        assembly = fasta.toString().replaceAll(/\.\w+/,"")
+
+        """
+        build_biscuit_QC_assets.pl -r $fasta -o ${assembly}_assets
```

Comment:
Yes, it has to do with `samtools 1.9` instead of `samtools 1.10`, this is the problem I was refering to here: https://github.com/nf-core/methylseq/pull/186#discussion_r580174146